### PR TITLE
Adding General Motors

### PR DESCRIPTION
**Pull request by johntheholman**
- Adds gm.com as a site that offers Multi-Factor Authentication
- Methods: Email, SMS and Software Auth
- This is a vehicle company so I have it in the Other and Transport categories because there is no "Vehicle" category.

### CODE REVIEW

**Comment by phallobst** [comment]
Here's an SVG version of the logo:
gm com

File: entries/g/gm.com.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "img": "gm.com.png",
+    "tfa": [
+      "sms",
+      "email",
+      "totp"
+    ],
+    "notes": "Automatically turned on for email. You are only allowed to have one method turned on at a time.",
+    "categories": [
+      "transport",
+      "other"
```
Comment (hkamran80) [inline entries/g/gm.com.json]:
```suggestion
      "transport"
```

I would place it only in `transport`

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/g/gm.com.json (hunk 1 above)
Comment (johntheholman) [reply entries/g/gm.com.json]: Thanks @hkamran80. I have updated the category to `transport` only.

**Comment by hkamran80** [comment]
Can you provide evidence of 2FA? It can be the account settings, the login flow, or the setup flow.

**Review by hkamran80 — COMMENTED** [review]
Can you provide evidence of 2FA? It can be the account settings, the login flow, or the setup flow.

**Comment by johntheholman** [comment]
They do not have any specific documentation but here are screenshots from the site
AEA6D7EC-229D-42BD-93B7-00A44100ABAE
EC1EB69F-05F3-4609-B997-C565E45CFFD8

**Comment by hkamran80** [comment]
Thank you!

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**